packages/orbit-design-tokens/src/dictionary/definitions/component/heading/heading-font-size.json

```diff
@@ -13,16 +13,22 @@
           "value": 22
         }
       },
-      "title-1": {
+      "title-0": {
         "font-size": {
           "type": "size",
           "value": 28
         }
       },
+      "title-1": {
+        "font-size": {
+          "type": "size",
+          "value": 24
```

Comment:
You mean because values change? I don't know, but I don't think it is. I feel the tokens are the abstraction behind values and a change to those values should not result in a breaking change (if we don't consider visual breaking changes). In the past we've been changing values of already existing tokens and not mark them as BC. Or am I missing a detail here?

Reply:
Yes, I was referring to the value change uniquely. But if you already had this practice in the past I guess it's OK. We have other related BCs in any case.

Reply:
> if we don't consider visual breaking changes

I guess that's the key.

Nothing will break with the value change but there might be unexpected visual changes.

Reply:
Yes, we don't have a consistent procedure for visual breaking changes… In this case, like you say, we will thoroughly mention this in the consequent breaking changes and affected components, so I feel it's ok to leave it as is